lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -1625,6 +1625,8 @@ def cmd_sessions(*args)
             elsif session.type == 'shell' || session.type == 'powershell'
               output = session.shell_command(cmd)
               print_line(output) if output
+            elsif session.type == 'MSSQL' || session.type == 'PostgreSQL' || session.type == 'MySQL'
```

Comment:
```suggestion
            elsif session.type == 'mssql' || session.type == 'postgresql' || session.type == 'mysql'
```
I believe we should be using all lowercase for these now

Reply:
we could also maybe change this to a case statement? probably look a bit cleaner overall

interesting how the other session types have the printing being done here but presumably the SQL session types are printing the output themselves? is that something we should consider changing to line up better with existing sessions? 🤔